test/lib/autosetup_helpers_test.rb

```diff
@@ -242,6 +242,47 @@ class DummyClient < Yast::Client
 
   end
 
+  describe "#autosetup_firewall" do
+    let(:profile) { firewall_section }
```

Comment:
Um, so the profile is the same as the firewall section? This looks a bit strange, I'd expect something like

```ruby
let(:profile) { { "firewall" => firewall_section } }
let(:firewall_section) { { "default_zone" => "external" } }
```

Reply:
ok